src/store/helpers.ts

```diff
@@ -1,5 +1,12 @@
-import type { V1SourceResponse } from './provisioningApi';
+import type { V1ListSourceResponse } from './provisioningApi';
+
+export type OldSourceResponse = {
+  id?: string;
+  name?: string;
+  source_type_id?: string;
+  uid?: string;
+};
 
 export const extractProvisioningList = (
```

Comment:
This is still keeping the backwards compatibility, but we can just wait with merging this a bit (few prod pushes) and then drop the backwards compatibility completely.

Reply:
It's been a while and everything seems to work fine, dropping the backwards compatibility :)